src/chrome/content/rules/Infobae.com.xml

```diff
@@ -0,0 +1,16 @@
+<!--
+	Nonfunctional hosts in *.infobae.com:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Infobae.com">
+	<target host="www.infobae.com" />
```

Comment:
You can add back `<target host="infobae.com" />` and make the addition in the comment below